main.sh

```diff
@@ -185,6 +185,21 @@ if [ -z "$OUTPUT_PATH" ]; then
   fi
 fi
 
+if ! command -v pciutils i2c-tools mesa-utils smartmontools dmidecode >/dev/null 2>&1; then
```

Comment:
It's better to not have duplicate code, use the code you have already written
```suggestion
if [[ ! ./check_dependencies.sh ]]; then
```